### DISCUSSION TITLE
🚔 7/1 1차 회의 겸 정리

### DISCUSSION THREAD

**Discussion by yukyunglee** [yukyunglee/Awesome-Dialogue-State-Tracking#5] [Ideas]
* 대분류는 Dataset ~ model로 결정
* Few shot/ zero shot / meta/ transfer leraning을 위한 카테고리 추가
* 나머지는 (일단) etc로
* 좀더 깔끔한 Format은 계속 찾아보기
* Table of contents는 8일 이후에 결정하기
* 시간나면 설명 조금 더 추가하기 (분류? 정도로 마인드맵이나 카테고리 정리하는 그림 있으면 좋을듯 / 강의때 나눈 분류도 좋았음 !)
* Score는 포함하지 않는 방향으로 설정하기

* DST 범위 : Task 사이에 Dialogue State tracking을 진행한다면 모두 DST로 분류
* venue의 범위 : 일단 toptier 위주로 하되, 너무 생소한 저널/ 컨퍼는 제외하는 방향으로 
* Dataset list는 계속 남겨두기
* 2021 paper 계속 추가하기 
* DSTC (AAAI special session paper -> introduction paper는 제외 ! / 근데 다시 생각해보니 넣어야 하나 .. . . . .. ? 갑자기 고민..)

**Comment by Kyumin-Park** [comment]
DST 범위 : Task 사이에 Dialogue State tracking을 진행한다면 모두 DST로 분류 - 이부분은 어느정도는 heuristic하게? 각자 판단에 맡기는 쪽으로 해도 될꺼같아요 (본인이 맞다 생각하면 넣는 정도로)
DSTC (AAAI special session paper -> introduction paper는 제외 ! / 근데 다시 생각해보니 넣어야 하나 .. . . . .. ? 갑자기 고민..) - 데이터셋에 없는 (DSTC1/2/3/8(SGD)이 아닌) 대회는 분류 옆에 넣거나 분류에 링크 걸어도 될꺼같아요

예시:

# [DSTC9](https://drive.google.com/file/d/0Bx4CHsnRHDmJLWVObHBtcnBYVFA1dUVCY2ZwRUFvMWx0clVj/view?resourcekey=0-oE0b2hOgTRZmKLrpivlR5A)

**Reply by yukyunglee** [reply]
오 좋습니다 저렇게 링크 넣어둘게요 ! 👍🏻
